contracts/modules/Checkpoint/Voting/APLCR/AdvancedPLCRVotingCheckpoint.sol

```diff
@@ -485,6 +486,7 @@ contract AdvancedPLCRVotingCheckpoint is AdvancedPLCRVotingCheckpointStorage, Vo
             }
             choiceCount = temp;
         }
+        require(totalChoiceWeight <= weight);
```

Comment:
Is this change due to moving some of the voting stuff to a library, or a separate bug you found?